ansible/roles/test/tasks/pfc_wd.yml

```diff
@@ -114,6 +114,14 @@
 
 - debug: msg="{{select_test_ports}}"
 
+- name: Disable logrotate during test run.
+  shell: sed -i 's/^/#/g' /etc/cron.d/logrotate
+  become: yes
+
+- name: Wait logrotate configuration change to take effect
+  pause:
+    seconds: 60
```

Comment:
any references to this 60 seconds number?

Reply:
60 seconds is current logrotate cron job configuration